x/oracle/keeper/grpc_query.go

```diff
@@ -77,12 +81,12 @@ func (q Keeper) Response(c context.Context, req *types.QueryResponseRequest) (*t
 	}
 	ctx := sdk.UnwrapSDKContext(c)
 
-	task, err := q.GetTask(ctx, req.Contract, req.Function)
+	task, err := q.GetTask(ctx, types.NewTaskID(req.Contract, req.Function))
```

Comment:
I wonder if we should refactor the task ID to be more general (I don't think the current task ID covers the new type well)

Reply:
The task ID's been redefined as actually `[]byte`. Do you have any idea about this?

Reply:
hmm I wonder if it can just be a uint64

Reply:
Oracle module could maintain and allocate a uint64-typed task id. 
But there may be "fast path" case where score response comes before the task request. In such a situation, it will be the operator that set the task id. So I guess at least the task id determination mechanism should be shared among the Oracle module and operator.